activemq-broker/src/main/java/org/apache/activemq/broker/BrokerService.java

```diff
@@ -2048,7 +2047,7 @@ protected void checkTmpStoreUsageLimits() throws Exception {
                 long maxJournalFileSize;
 
                 PListStore store = usage.getTempUsage().getStore();
-                if (store != null && store instanceof JournaledStore) {
+                if (store instanceof JournaledStore) {
```

Comment:
Same here, `store` can be `null` if temp usage is not set.

Reply:
Yes. But `instanceof` should handle the nullchecking, so the first check is redundant.

Reply:
Ah yeah, good catch. Thanks !